roles/zuul/files/jobs/dib.yaml

```diff
@@ -0,0 +1,23 @@
+- job:
+    name: 'dib-build'
+    node: 'ubuntu-xenial'
+    builders:
+      shell: |
+        virtualenv $HOME/dib-build
+        . $HOME/dib-build/bin/activate
+        pip install diskimage-builder
+
+        git clone https://github.com/BonnyCI/hoist.git /opt/hoist
```

Comment:
The bonnyci user isn't going to have access to create this directory. Need to sudo this or create it in ~